jdbc/src/main/java/software/amazon/timestream/jdbc/TimestreamSchemasResultSet.java

```diff
@@ -15,22 +15,100 @@
 package software.amazon.timestream.jdbc;
 
 import com.amazonaws.services.timestreamquery.model.ColumnInfo;
+import com.amazonaws.services.timestreamquery.model.Datum;
+import com.amazonaws.services.timestreamquery.model.Row;
+import com.google.common.base.Strings;
 import com.google.common.collect.ImmutableList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
 import java.util.List;
 
 /**
  * Result set to return an empty list of schemas in Timestream.
  */
-public class TimestreamSchemasResultSet extends TimestreamEmptyBaseResultSet {
+public class TimestreamSchemasResultSet extends TimestreamBaseResultSet {
+  private static final Logger LOGGER = LoggerFactory.getLogger(TimestreamDatabasesResultSet.class);
+  private static final Datum NULL_DATUM = new Datum().withNullValue(Boolean.TRUE);
   private static final List<ColumnInfo> COLUMNS = ImmutableList.of(
     TimestreamDataType.createColumnInfo(TimestreamDataType.VARCHAR, "TABLE_SCHEM"),
```

Comment:
Does order matter here? Wouldn't `TABLE_CATALOG` come before `TABLE_SCHEM`? This order doesn't matter since it is only used internally.

Reply:
yes I believe the order matters here. I was confused at first as well. According to Msft, in [`getSchemas()`](https://learn.microsoft.com/en-us/sql/connect/jdbc/reference/getschemas-method?view=sql-server-ver16), `TABLE_SCHEM` does come before `TABLE_CATALOG`.